src/autobridgecalculator.js

```diff
@@ -70,14 +73,17 @@ AutobridgeCalculator.prototype.calculate = function() {
       continue;
     }
 
-    if (legOneTakerGetsFunded.compareTo(legTwoTakerPaysFunded) > 0) {
+    //  using private fields for speed
```

Comment:
Yes, from_json is generally very slow and must not be used in inner loops. AutobridgeCalculator is slowest part of the code, so I'm trying to squeeze every last bit of performance out of it.
And, if you look into `OrderBookUtils` - I'm using private field of `Amount` there, because I really need a fast way to construct `Amount` object.